docs/includes/azure-container-app-naming.md

```diff
@@ -0,0 +1,3 @@
+When creating new Azure resources, it's important to follow the naming requirements. For Azure Container Apps, the name should be 2-32 characters long and consist of lowercase letters, numbers, and hyphens. The name should start with a letter and end with an alphanumeric character.
```

Comment:
```suggestion
When you create new Azure resources, it's important to follow the naming requirements. For Azure Container Apps, the name should be 2-32 characters long and consist of lowercase letters, numbers, and hyphens. The name should start with a letter and end with an alphanumeric character.
```

Reply:
Is *should be* strong enough here? Deployment will fail if these rules are followed, I think this should be *must be* or otherwise indicate that the deployment will fail if the rules aren't followed.